### PR TITLE
update string, update sleepy scugs

### PR DESCRIPTION
**Pull request by 6fears7**

### CODE REVIEW

File: Story/StoryMenu.cs [hunk 1]
```diff
@@ -398,7 +398,7 @@ private void CustomSlugcatSetup()
                 var btn = new SimplerButton(this, mainPage, SlugcatStats.getSlugcatName(slug), new Vector2(394, 515) - i * new Vector2(0, 38), new Vector2(110, 30));
                 btn.toggled = false;
                 mainPage.subObjects.Add(btn);
-
+                customSelectedSlugcat = slugList[0]; // default as Survivor so no sleep walking scugs
```
Comment (youbitchoc) [inline Story/StoryMenu.cs:401]: must this be in the loop? could we set the default value at initialization?

**Review by youbitchoc — CHANGES_REQUESTED** [review]

File: Story/StoryMenu.cs (hunk 1 above)
Comment (6fears7) [reply Story/StoryMenu.cs:401]: Sure could. Won't be back at it for another 36 hours or so, so if you wanna add those changes feel free. Else I'll get it then

Comment (youbitchoc) [reply Story/StoryMenu.cs]: not too happy with hardcoding it to White but the lobby menu relies on this anyway (arena hardcodes it too) so it should be fine

**Comment by youbitchoc** [comment]
lgtm
(there's some extra newlines here and there but am just being pedantic)

**Pull request merged**